open-api/rest-catalog-open-api.yaml

```diff
@@ -61,12 +65,26 @@ security:
   - OAuth2: [catalog]
   - BearerAuth: []
 
+# Capability tags describe optional functionality and are used to group endpoints together.
+# In order to support a particular capability, a server is required to implement all endpoints grouped under a particular tag.
+tags:
+  - name: tables
+    description: Required as part of table support
+  - name: views
+    description: Required as part of view support
+  - name: remote-signing
```

Comment:
if we consider this as a capability, do we also consider `credentials-vending` a capability? I think we should.

Reply:
I had this initially but I forgot why I removed it. Added it back

Reply:
@jackye1995 note that I called this `vended-credentials` to align with other places in the REST spec that also use `remote-signing` / `vended-credentials`(instead of `credential-vending`)